build.sbt

```diff
@@ -151,19 +154,40 @@ lazy val iam = project
                                 scalaTest   % Test)
   )
 
-lazy val shaclValidator = project
-  .in(file("modules/ld/shacl-validator"))
-  .dependsOn(types)
+lazy val indexTypes = project
+  .in(file("modules/index/types"))
   .settings(
-    name                := "shacl-validator",
-    moduleName          := "shacl-validator",
-    resolvers           += Resolver.bintrayRepo("bogdanromanx", "maven"),
-    libraryDependencies ++= Seq(journal, wesoSchema, catsCore, circeCore, circeParser % Test, scalaTest % Test)
+    name                := "commons-index-types",
+    moduleName          := "commons-index-types",
+    libraryDependencies ++= Seq(akkaHttp, catsCore, circeCore, scalaTest % Test, circeGenericExtras % Test)
+  )
+
+lazy val elasticClient = project
+  .in(file("modules/index/elastic-client"))
+  .dependsOn(http, indexTypes, test)
+  .settings(
+    name       := "elastic-client",
+    moduleName := "elastic-client",
+    libraryDependencies ++= Seq(
+      akkaStream,
+      circeCore,
+      circeOptics,
+      circeParser                % Test,
+      circeGenericExtras         % Test,
+      akkaTestKit                % Test,
+      scalaTest                  % Test,
+      "org.apache.logging.log4j" % "log4j-api" % log4jVersion % Test,
+      "org.apache.logging.log4j" % "log4j-core" % log4jVersion % Test,
+      "org.elasticsearch"        % "elasticsearch" % elasticSearchVersion % Test,
+      "org.elasticsearch.client" % "rest" % elasticSearchVersion % Test,
+      "org.elasticsearch.plugin" % "transport-netty3-client" % elasticSearchVersion % Test,
+      "org.apache.commons"       % "commons-io" % "1.3.2" % Test
```

Comment:
move the commons-io version top level; move dependency definitions top level in a separate block.